src/main/cpp/teleopcontrol/TeleopControlFunctions.h

```diff
@@ -27,13 +27,17 @@ class TeleopControlFunctions
 public:
     enum FUNCTION
     {
+        READY,
         UNKNOWN_FUNCTION,
```

Comment:
@witcpalek  Should UNKOWN_FUNCTION  be first? Or can this just be removed? I don't see anything at the end of the enum to signify the end.

Reply:
@Tan560 This used to be used for initialization, but I don't see any usages any more.   So, it could be removed, but if it is kept, it should be first to make reading/understanding better.